src/Transactions/types.ts

```diff
@@ -23,16 +23,26 @@ export interface BlockFrag {
     epochNo: number;
     slotNo: number;
 }
+
+export interface Asset {
+    assetId: string,
+    policyId: null | string,
+    name: null | string,
+    amount: number,
+}
+
 export interface TransInputFrag {
     address: string;
     amount: string;
     id: string;
     index: number;
     txHash: string;
+    assets: null | [Asset]
 }
 export interface TransOutputFrag {
     address: string;
     amount: string;
+    assets: null | [Asset]
```

Comment:
Probably it's better to return empty array instead of null to avoid having to do null-checks everywhere